packages/wallet-sdk/src/sign/SignRequestHandler.ts

```diff
@@ -20,34 +16,14 @@ export class SignRequestHandler {
     this.tryRestoringSignerFromPersistedType();
   }
 
-  async handleRequest(request: RequestArguments, accounts: AddressString[]) {
-    try {
-      if (request.method === 'eth_requestAccounts') {
-        if (accounts.length > 0) return accounts;
-        return await this.requestAccounts();
-      }
-
-      const signer = await this.useSigner();
-      if (accounts.length <= 0) {
-        throw standardErrors.provider.unauthorized(
-          "Must call 'eth_requestAccounts' before other methods"
-        );
-      }
-
-      return await signer.request(request);
-    } catch (err) {
-      if ((err as { code?: unknown })?.code === standardErrorCodes.provider.unauthorized) {
```

Comment:
this unauthorized is for handling a few edge cases, including where user disconnected from dapp side, but SCW is still at "connected" state. think we cant remove it?